foundation/foundation-mda/src/main/resources/templates/deployment/spark-infrastructure/v2/spark.infrastructure.values.yaml.vm

```diff
@@ -12,11 +12,11 @@ aissemble-thrift-server-chart:
   deployment:
     envFromSecret:
       AWS_ACCESS_KEY_ID:
-        secretName: aws-s3-credentials
-        key: aws_access_key_id
+        secretName: remote-auth-config
```

Comment:
I: It was brought to my attention that we publish this secret as part of the localstack chart, so we use it for out of the box functionality and as a model of good practice.